cowamm/.sqlfluff

```diff
@@ -10,3 +10,4 @@ results=final_results_per_solver,cow_surplus_per_batch
 cow_budget=30000
 number_of_pools=1000
 competitor_end_time='2024-08-27 00:00:00'
+frequence='week'
```

Comment:
nit: on https://dune.com/cowprotocol/cowswap-high-level-metrics-dashboard we call it `aggregate_by`, maybe we can keep this for consistency (I also find it more clear as frequency makes me think of how often something is exectued)